torch/autograd/forward_ad.py

```diff
@@ -44,14 +41,16 @@ def exit_dual_level(*, level=None):
     _current_level = level - 1
 
 def make_dual(tensor, tangent, *, level=None):
-    r"""Function that creates a "dual object" that can be used to compute forward AD gradients
-    based on the given Tensor and its tangent. It returns a new Tensor that shares memory with
-    :attr:`tensor` and the :attr:`tangent` is used as-is.
+    r"""Associates a tensor value with a forward gradient, the tangent, to create a
+    "dual tensor", which is used used to compute forward AD gradients. The forward gradient,
+    :attr:`tangent`, must have the same storage layout as the tensor value, :attr:`tensor`.
+    The result is a new tensor aliased to :attr:`tensor` with :attr:`tangent` embedded
```

Comment:
And the end here "with :attr:`tangent` embedded as an attribute as-is." should be "with :attr:`tangent` embedded as an attribute as-is if it has the same storage layout or cloned otherwise".